core/test/cosmos/transactions_test.cpp

```diff
@@ -49,7 +49,41 @@
 using namespace ledger::testing::cosmos;
 using namespace ledger::core;
 
-class CosmosTransactionTest : public BaseFixture {};
+class CosmosTransactionTest : public BaseFixture {
+public:
+ void SetUp() override
+ {
+     BaseFixture::SetUp();
+#ifdef PG_SUPPORT
+     const bool usePostgreSQL = true;
+     auto poolConfig = DynamicObject::newInstance();
+     poolConfig->putString(api::PoolConfiguration::DATABASE_NAME, "postgres://localhost:5432/test_db");
+     pool = newDefaultPool("postgres", "", poolConfig, usePostgreSQL);
+#else
+     pool = newDefaultPool();
+#endif
+     backend->enableQueryLogging(true);
+ }
+
+ void setupTest(std::shared_ptr<CosmosLikeAccount> &account,
+                std::shared_ptr<CosmosLikeWallet> &wallet)
+ {
+     auto configuration = DynamicObject::newInstance();
+     configuration->putString(
+         api::Configuration::KEYCHAIN_DERIVATION_SCHEME,
+         "44'/<coin_type>'/<account>'/<node>/<address>");
+     wallet = std::dynamic_pointer_cast<CosmosLikeWallet>(
+         wait(pool->createWallet("e847815f-488a-4301-b67c-378a5e9c8a61", "atom", configuration)));
+
+     auto accountInfo = wait(wallet->getNextAccountCreationInfo());
+     EXPECT_EQ(accountInfo.index, 0);
+     accountInfo.publicKeys.push_back(hex::toByteArray(DEFAULT_HEX_PUB_KEY));
+
+     account = createCosmosLikeAccount(wallet, accountInfo.index, accountInfo);
+ }
+
+ std::shared_ptr<WalletPool> pool;
+};
```

Comment:
Is this required for this test suite? I removed it as it seemed unused here.

Reply:
Looks like unused right. I'll just remove the fixture altogether then